package.json

```diff
@@ -199,6 +203,11 @@
                     "group": "primary@2"
                 },
                 {
+                    "command": "ev3devBrowser.deviceTreeItem.printSysinfo",
+                    "when": "view == ev3devBrowser && viewItem == ev3devBrowser.device.connected",
+                    "group": "primary@2"
```

Comment:
The number after `@` needs to be changed to determine the order of the menu items. For example, to have this after the screenshot item, this should be `@3`.